src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java

```diff
@@ -137,15 +146,24 @@ public ElasticsearchClusterStats getClusterStats() {
         );
     }
 
-    private Map<String, NodeInfo> fetchNodeInfos() {
-        ClusterAdminClient adminClient = this.client.admin().cluster();
-        NodesInfoResponse nodesInfoResponse = adminClient.nodesInfo(adminClient.prepareNodesInfo().request()).actionGet();
-        return nodesInfoResponse.getNodesMap();
-    }
-
-    private Map<String, NodeStats> fetchNodeStats() {
-        ClusterAdminClient adminClient = this.client.admin().cluster();
-        NodesStatsResponse nodesStatsResponse = adminClient.nodesStats(adminClient.prepareNodesStats().request()).actionGet();
-        return nodesStatsResponse.getNodesMap();
+    private JsonObject fetchNodeInfos() {
+        final NodesInfo.Builder requestBuilder = new NodesInfo.Builder()
+                .withHttp()
+                .withJvm()
+                .withNetwork()
+                .withOs()
+                .withPlugins()
+                .withProcess()
+                .withSettings()
+                .withThreadPool()
+                .withTransport();
+        try {
+            final JestResult result = jestClient.execute(requestBuilder.build());
+            return Optional.of(result.getJsonObject())
+                    .map(json -> GsonUtils.asJsonObject(json.get("nodes")))
+                    .orElse(null);
+        } catch (IOException e) {
+            return null;
```

Comment:
The idea was that returning `null` would allow the rest of the code to at least return partial results from the other calls instead of the exception bubbling up canceling everything.